packages/apps/src/Menu/index.tsx

```diff
@@ -44,6 +42,8 @@ function logDisabled (route: string, message: string): void {
 }
 
 function checkVisible (name: string, { api, isApiConnected, isApiReady }: ApiProps, hasAccounts: boolean, hasSudo: boolean, { isHidden, needsAccounts, needsApi, needsSudo }: Route['display']): boolean {
+  if (name === 'settings') return false;
```

Comment:
This is very problematic since we can never move settings to a menu item. (Like is planned, since we actually want to split meta alongside)

As shows in the actual functional views as well, this is also problematic.